docs/General/Manuscripts/index.md

```diff
@@ -1,3 +1,103 @@
+| :---- |
+
+# Response to Reviewers
+
+We thank the reviewers for their response and constructive comments for our manuscript titled “xxx” (Manuscript\_ID\_yyy). Based on the reviewers’ feedback, we have made substantial changes to the revised manuscript. Please find our response to the reviewers’ comments below. The reviewers’ comments are in black and our responses are in *blue*. 
+
+Examples of responses:
+
+| Reviewer \#1: This is a well written manuscript presenting artificial intelligence approaches to define bimodal gene expression as a tool to determine drug sensitivity in cancer. The premise of the work is that bimodal gene assessment would be a superior predictor of drug sensitivity. This work could deeply influence drug development and enhance the preclinical target validation in a comprehensive and robust way. The authors present extensive data demonstrating the role of AI in assessing bimodal gene expression as a comprehensive biomarker strategy. There are many interesting findings in this manuscript, but there are several areas of uncertainty and statements that should be addressed to place these findings in context of current and past clinical drug development and its relevance. *We thank the reviewer for her/his positive assessment of our manuscript. We have added our response to each comment below.*  Introduction: 1\) The introduction is somewhat unclear, are the authors focusing on gene expression as predictive markers of drug sensitivity assessment tools or prognostic markers as some of the examples that listed may suggest? *The focus of the manuscript is on genes with bimodal expression as candidate predictive biomarkers of drug sensitivity. The examples showcase the usefulness of bimodal genes in various biomarker settings as a motivation for their relevance. We have enhanced the relevant Introduction section to remove any confusion. Revised manuscript \- Section : For example, estrogen receptor (ESR1) bimodal expression defines two biological states within breast cancer patients. These states have been used to stratify breast cancer patients into the clinically-relevant subtypes (ER+/-) and derive treatment decisions. Another example in cancer genomics is the use of 73 bimodal genes within ovarian cancer to define molecular subtypes with distinct survival rate 12\. We also have shown that epithelial-to-mesenchymal transition (EMT) related genes were found to be bimodal pan-cancer and predictive of response to statin class of drugs 13\.* |
+| :---- |
+
```

Comment:
Colour code response and reviewer comment so that it is obvious to BHK and group